modules/RTCStats/RTCStats.ts

```diff
@@ -63,6 +64,56 @@ class RTCStats {
         this._initialized = true;
     }
 
+    /**
+     * A JitsiConnection instance is created before the conference is joined, so even though
+     * we don't have any conference specific data yet, we can initialize the trace module and
+     * send any logs that might of otherwise be missed if case an error occurs between the connection
+     * and conference initialization.
+     */
+    attachToConnection(initConfig: IRTCStatsConfiguration) {
```

Comment:
It reads weird that we call this attachToConnection and yet the connection is not passed around.

This is not the case in attachToConference, which does take a conference object.